InternshipApp/InternshipApp/Screens/CreateTaskView/CreateTaskView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  CreateTaskView.swift
+//  InternshipApp
+//
+//  Created by Sead Mašetić on 12. 8. 2024..
+//
+
+import SwiftUI
+
+struct CreateTaskView: View {
+    
+    @State private var taskName: String = ""
```

Comment:
If "taskViewModel" is created, then all of these properties should be inside of it, and you should access them through that viewModel